src/liquid/components/ld-tooltip/ld-tooltip.tsx

```diff
@@ -302,6 +314,20 @@ export class LdTooltip {
     }
   }
 
+  // This listener is required for tooltips which open via right click.
+  @Listen('ldtooltipopen', {
+    target: 'window',
+  })
+  handleContextMenuOutside(ev: CustomEvent) {
+    if (
+      this.visible &&
+      !ev.composedPath().includes(this.el) &&
+      !ev.composedPath().includes(this.tooltipRef)
+    ) {
+      this.hideTooltip()
+    }
+  }
```

Comment:
I don't see why this is now necessary, when tooltips didn't need it without the `oncontextmenu` event handler. Please explain.